client/trino-cli/src/main/java/io/trino/cli/FormatUtils.java

```diff
@@ -115,7 +115,7 @@ else if (longForm) {
     public static String formatDataRate(DataSize dataSize, Duration duration, boolean longForm)
     {
         long rate = Math.round(dataSize.toBytes() / duration.getValue(SECONDS));
-        if (Double.isNaN(rate) || Double.isInfinite(rate)) {
+        if (rate == Long.MAX_VALUE) {
```

Comment:
`Math.round` returns `Long.MAX_VALUE` for positive infinity and `Long.MIN_VALUE` for negative infinity. `Double.NaN` returns `0`. The check looks correct to me since `dataSize.toBytes()` can only return positive values, making negative infinity impossible.

Reply:
The check against `Long.MAX_VALUE` preserves the intended semantics (and mirrors those of `formatCountRate()`) - if the rate calculation results in 'infinity', then set it to zero.

Reply:
Are we printing "oh, so big a rate that I can't handle it" and "nothing is happening" the same way?

Reply:
Yes - this is how `formatCountRate` works. I'm merely fixing the check here so it works as intended.

Reply:
I'm pretty sure the only way to get infinity (and thus max long) is if the duration is effectively zero.  In that case I think printing a zero rate is fine.  It will quickly update to a real value.